Core/scripts/dirac-install.py

```diff
@@ -2148,6 +2148,20 @@ def createBashrc():
       # Add the lines required for fork support for xrootd
       lines.extend(['# Fork support for xrootd',
                     'export XRD_RUNFORKHANDLER=1'])
+
+      # Add possible DIRAC environment variables
+      lines.append('')
+      lines.append('# before enabling any of these variables, please see the documentation ')
+      lines.append('# https://dirac.readthedocs.io/en/latest/AdministratorGuide/' +
+                   'ServerInstallations/environment_variable_configuration.html')
+      lines.append('# export DIRAC_DEBUG_DENCODE_CALLSTACK=1')
+      lines.append('# export DIRAC_DEBUG_STOMP=1')
+      lines.append('# export DIRAC_DEPRECATED_FAIL=1')
+      lines.append('# export DIRAC_GFAL_GRIDFTP_SESSION_REUSE=true')
+      lines.append('# export DIRAC_USE_M2CRYPTO=true')
+      lines.append('# export DIRAC_USE_NEWTHREADPOOL=yes')
+      lines.append('# export DIRAC_VOMSES=/etc/grid-security/vomsdir')
```

Comment:
In `VOMS.py` this is `/etc/grid-security/vomses`

Reply:
On lxplus the vomses are in this folder "/etc/grid-security/vomsdir".

Reply:
@atsareg do you remember what's correct?

Reply:
I think in DIRAC this it is actually `$DIRAC/etc/grid-security/vomses` and the correct value is where ever someone wants to put their VOMS files. I could also put `/path/to/vomses`.

Reply:
Please put `$DIRAC/etc/grid-security/vomses` for consistency.